src/main/java/ch/njol/skript/classes/data/DefaultConverters.java

```diff
@@ -198,6 +199,9 @@ public DefaultConverters() {}
 
 		Converters.registerConverter(String.class, World.class, Bukkit::getWorld);
 
+		// LootContext - Location
```

Comment:
You could register a String -> Loot table converter, then you'd be able to get rid of the 'table from string' expression and could just use `set loot table of {blah} to "minecraft:end_city_chest"` (or whatever the syntax is) in your code.